spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -81,9 +90,10 @@ class { 'foreman::cli':
 
       it_behaves_like 'hammer'
 
-      package_name = "rubygem-hammer_cli_katello"
-      describe package(package_name) do
-        it { is_expected.to be_installed }
+      ['katello', 'foreman_virt_who_configure'].each do |plugin|
```

Comment:
Because I searched for the virt_who_configure package name and I just noticed we have a lot of packages with tags in them: http://koji.katello.org/koji/search?terms=*.el7&type=package&match=glob
http://koji.katello.org/koji/search?terms=*.el8&type=package&match=glob